lib/mable_etl/control.rb

```diff
@@ -47,5 +60,15 @@ def transform
     def load
       @load ||= MableEtl::Loaders::LoaderFactory.for(params).load
     end
+
+    def log_result(result)
```

Comment:
Needs test cases for 

* when extracting logger receives info with result.message
* when extracting logger receives error with result.message
* when transforming logger receives info with result.message
* when transforming logger receives error with result.message